ansible/molecule/ec2_gpu/converge.yaml

```diff
@@ -19,6 +19,17 @@
       set_fact:
         ansible_python_interpreter: /opt/bin/python
       when: flatcar_bootstrapped.rc == 0
+
+- hosts: all
+  name: Hack to pass vars
+  become: true
+  # TODO: hack to pass vars easily for now
+  pre_tasks:
+    - include_vars: ../../vars/centos/centos-7.yaml
+      when: ansible_os_family == "RedHat"
+    - include_vars: ../../vars/flatcar/flatcar.yaml
+      when: ansible_os_family == "Flatcar"
+
```

Comment:
Does this actually work? I haven't had much success using this.

Reply:
It does, this includes the flatcar specific overrides. Otherwise things like ie. `nvidia_driver_version`would not be defined for Flatcar.

The centos-7 variant `../../vars/centos/centos-7.yaml`  however is empty. Nothing good coming out of that.